### PR TITLE
Add Docker support

### PR DESCRIPTION
**Pull request by lijingpeng**
Add Docker support and add instructions in README.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+RUN pip install --trusted-host pypi.douban.com -i http://pypi.douban.com/simple/ -U pip
+RUN pip install --trusted-host pypi.douban.com -i http://pypi.douban.com/simple/ -U Pillow
+# RUN pip install --trusted-host pypi.douban.com -i http://pypi.douban.com/simple/ -U pyopenssl ndg-httpsclient pyasn1
+
+# Pre-trained network, too large, use local volume instead.
+RUN apt-get install -y wget
+#RUN wget http://www.vlfeat.org/matconvnet/models/beta16/imagenet-vgg-verydeep-19.mat
+#RUN mv imagenet-vgg-verydeep-19.mat neural-style
+
+CMD ["/run_jupyter.sh"]
```
Comment (anishathalye) [inline Dockerfile]: did you not check this file into Git?

File: README.md [hunk 2]
```diff
@@ -57,6 +57,33 @@ style blend weight of 0.2:
...
+```
+Then you can run neural-style code with jupyter notebook in http://localhost:8888/
+
+#### Docker run in command
+```
+docker run -it -p 8888:8888 -v /Users/you:/notebooks/neural-style-mat docker_neural_style /bin/bash
+```
+```
+python neural_style.py --content examples/1-content.jpg --styles examples/1-style.jpg --output examples/myoutput.jpg --network ../neura
+l-style-mat/imagenet-vgg-verydeep-19.mat
```
Comment (anishathalye) [inline README.md]: accidental line break?

Comment (lijingpeng) [inline README.md]: this line break should be moved. it's a typo.

**Comment by lijingpeng** [comment]
Sadly original docker cannot access to GPU on local computer, BUT nvidia-docker CAN DO IT. Since I don't have any GPU based computer, I cannot test it. And I think CPU based docker is also useful for new deep learning starters because:
1. Easy to use. don't have to establish a neural networks environment with dependence Hell. Only docker is enough.
2. Easy to transfer it to any server computer or AWS/Google Cloud through docker. Save time to focus on the algorithm of deep learning instead of installing softwares on different environments.

**Comment by anishathalye** [comment]
I think this is unnecessary / orthogonal from the neural style implementation. If people want to use Docker or whatever, they can figure it out themselves.

**Pull request closed without merging**